frontend/app/mocks/db.ts

```diff
@@ -104,6 +110,21 @@ for (let i = 0; i < numberOfLetters; i++) {
   });
 }
 
+// seed letter type list
+db.letterType.create({
+  id: faker.string.alphanumeric(10),
+  code: 'ACC',
+  nameEn: 'Accepted',
+  nameFr: '(FR) Accepted',
+});
+
+db.letterType.create({
+  id: faker.string.alphanumeric(10),
```

Comment:
You don't need to set the `id` since it will use `faker.string.uuid` as default.

```
id: primaryKey(faker.string.uuid),
```